.github/actions/spelling/advice.md

```diff
@@ -8,20 +9,42 @@ By default the command suggestion will generate a file named based on your commi
 :warning: The command is written for posix shells. You can copy the contents of each `perl` command excluding the outer `'` marks and dropping any `'"`/`"'` quotation mark pairs into a file and then run `perl file.pl` from the root of the repository to run the code. Alternatively, you can manually insert the items...
 
 If the listed items are:
+
 * ... **misspelled**, then please *correct* them instead of using the command.
-* ... *names*, please add them to `.github/actions/spelling/dictionary/names.txt`.
-* ... APIs, you can add them to a file in `.github/actions/spelling/dictionary/`.
+* ... *names*, please add them to `.github/actions/spelling/allow/names.txt`.
+* ... APIs, you can add them to a file in `.github/actions/spelling/allow/`.
 * ... just things you're using, please add them to an appropriate file in `.github/actions/spelling/expect/`.
 * ... tokens you only need in one place and shouldn't *generally be used*, you can add an item in an appropriate file in `.github/actions/spelling/patterns/`.
 
 See the `README.md` in each directory for more information.
 
 :microscope: You can test your commits **without** *appending* to a PR by creating a new branch with that extra change and pushing it to your fork. The [check-spelling](https://github.com/marketplace/actions/check-spelling) action will run in response to your **push** -- it doesn't require an open pull request. By using such a branch, you can limit the number of typos your peers see you make. :wink:
 
-:clamp: If you see a bunch of garbage and it relates to a binary-ish string, please add a file path to the `.github/actions/spelling/excludes.txt` file instead of just accepting the garbage.
+<details><summary>:clamp: If you see a bunch of garbage</summary>
+
+If it relates to a ...
+<details><summary>well-formed pattern</summary>
+
+See if there's a [pattern](https://github.com/check-spelling/check-spelling/wiki/Configuration-Examples:-patterns) that would match it.
+
+If not, try writing one and adding it to the `patterns.txt` file.
```

Comment:
This should probably say:
```suggestion
If not, try writing one and adding it to a `patterns/{file}.txt`.
```

The goal is to guide people to the patterns collection.